src/Component/Filter/NumericRangeFilter.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Grid\Filter;
+
+use Sylius\Component\Grid\Data\DataSourceInterface;
+use Sylius\Component\Grid\Filtering\FilterInterface;
+
+final class NumericRangeFilter implements FilterInterface
+{
+    public const DEFAULT_SCALE = 0;
+
+    public const DEFAULT_ROUNDING_MODE = \NumberFormatter::ROUND_HALFUP;
+
+    public const DEFAULT_INCLUSIVE_FROM = true;
+
+    public const DEFAULT_INCLUSIVE_TO = true;
+
+    /** @param array<array-key, string> $data */
+    public function apply(DataSourceInterface $dataSource, string $name, $data, array $options): void
+    {
+        if (empty($data)) {
+            return;
+        }
+
+        $field = (string) ($options['field'] ?? $name);
+        $scale = (int) ($options['scale'] ?? self::DEFAULT_SCALE);
+        $mode = (int) ($options['rounding_mode'] ?? self::DEFAULT_ROUNDING_MODE);
+
+        $greaterThan = $this->getDataValue($data, 'greaterThan');
+        $lessThan = $this->getDataValue($data, 'lessThan');
+
+        $expressionBuilder = $dataSource->getExpressionBuilder();
+
+        if ('' !== $greaterThan) {
+            $inclusive = (bool) ($options['inclusive_from'] ?? self::DEFAULT_INCLUSIVE_FROM);
```

Comment:
This statement can be moved up, outside the if statement, and the one on line 57 can be removed.

Reply:
Unfortunately, but it cannot be removed due to the fact that they are two different fields `inclusive_from` and `inclusive_to`.

Therefore, moving this `statement` higher in this case will not make sense.
Or did I miss something :) ?

Reply:
Right, my mistake.